src/dom/at-class.ts

```diff
@@ -0,0 +1,190 @@
+import {element} from 'angular';
+import {DomWatcher} from '../dom/dom-watcher';
+
+/**
+ * A function that returns true or false.
+ */
+export type AtClassEvaluationCallBack = (...args: any[]) => boolean;
+
+export interface AtClassConfig {
+  /**
+   * The root element to observe and to scope at class functionality to.
+   */
+  element: HTMLElement;
+
+  /**
+   * A list of tuple [name, callback] to check on each resize.
+   */
+  conditions: [string, AtClassEvaluationCallBack][];
```

Comment:
I think tsstyle would wants you to use `Array<[string, AtClassEvaluationCallback]>` here.

https://google.github.io/styleguide/tsguide.html#arrayt-type

Reply:
Done